include/boost/geometry/algorithms/detail/buffer/turn_in_original_visitor.hpp

```diff
@@ -226,13 +231,13 @@ class turn_in_original_visitor
             return true;
         }
 
-        if (geometry::disjoint(turn.robust_point, original.m_box))
+        if (geometry::disjoint(turn.point, original.m_box))
```

Comment:
Just for completness, here the strategy is needed as well. I'm ok with merging it though.